lib/iris/tests/integration/test_grib_load.py

```diff
@@ -131,6 +133,7 @@ def test_reduced_ll(self):
             ("GRIB", "reduced", "reduced_ll.grib1")))
         self.assertCML(cube, ("grib_load", "reduced_ll_grib1.cml"))
 
+    @skip('iris-grib is currently causing failures')
     def test_reduced_gg(self):
```

Comment:
This failure seems to have been introduced by the latest verion of iris-grib. The points of one of the dimensions is reading [0.] rather than [~2.] .

Reply:
This is fixed by the changes suggested in https://github.com/SciTools/iris-grib/issues/150